git-quick-stats

```diff
@@ -1124,7 +1124,7 @@ while [[ "${opt}" != "" ]]; do
            while [[ -z "${newDate}" ]]; do
                read -r -p "Since what date? " newDate
```

Comment:
`date -d` is amazingly versatile. Sometimes so versatile that is can be a bit of a landmine when we start adding international users into the mix and consider all valid/invalid possibilities based on locale.

Let's provide a small prompt after this showing some basic examples, and the user can then extrapolate this based off their own local date and time format. I'm thinking something like:

`Since what date? (e.g. 2024-09-18, 18 September 2024, next Monday): `

Thoughts? This same comment applies for the non-interactive mode above, so both areas would need to be changed for consistency.

Reply:
Makes sense, I'll do that.